models/workday__organization_overview.sql

```diff
@@ -1,4 +1,4 @@
-with organization_data as (
+with organization_data as ( 
```

Comment:
This end model is defined as:

> Each record represents an organization, enhanced with role details.

However, it seems the grain is organization+role as opposed to just organization. Should we consider renaming this end model or updating the documentation to clearly detail the grain. What are your thoughts?

Reply:
Let's consider renaming this to be `workday__role_overview` as this is more representative of the grain of the data.

Reply:
Renamed.

Reply:
I decided to keep this as `workday__organization_overview` (no one is really going to look at role too closely in comparison to organization) and improved the documentation. 

I also realized doing just role and organization as the grain for the `organziation_overview` provides minimal insight as an end model. I did want to find a way to tie it back to the `employee_overview` somehow so customers can join back to those tables if they want to do their own transformations. So I also added in `worker_id` and `position_id`  to the grain, by adding a new staging model which ties worker and position back to organization (I noticed that in the new history tables there was a `worker_position_organization_history`  table I should have brought into begin with).